views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,27 @@
+export const siteTooltipContent = (siteData) => {
+  const tooltipData = []
+  const reserved = 'N/A'
+  const totalLabel = 'Total'
+  let siteTotalCount = 0
+  let siteTotalPercent = 0
+
+  siteData
+    .filter(({ name }) => name !== reserved)
+    .forEach(({ name: valueName, payload }) => {
+      if (valueName !== reserved) {
+        const valuePercent = payload.percentages[valueName].toFixed(0)
+        const valueColumn = `${payload.counts[valueName]}/(${valuePercent}%)`
+        const rowValue = { labelColumn: valueName, valueColumn }
+        siteTotalCount = siteTotalCount + payload.counts[valueName]
+        siteTotalPercent = siteTotalPercent + +valuePercent
+        tooltipData.push(rowValue)
+      }
+    })
+
+  tooltipData.push({
+    labelColumn: totalLabel,
+    valueColumn: `${siteTotalCount}/(${siteTotalPercent}%)`,
```

Comment:
I think we should use our percent formatter for this and the other tooltip rows.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/82a8267770a707eedb1efe2f1cc9193d667b1d12